src/flight-search/filtering.js

```diff
@@ -31,8 +31,17 @@ function filterByProviders(trip, providerFilter) {
 
 function isFareMatchProviderType (fare, providerTypes) {
   if (!providerTypes) return true;
-  var isMatchTypeInstant = (fare.provider.instant && providerTypes.includes('instant'));
-  return isMatchTypeInstant || (providerTypes.includes(fare.provider.type));
+  return _isFacilitatedBooking(fare, providerTypes) || _isDirectBooking(fare, providerTypes);
```

Comment:
looks like if we want to add another provider type in the filter, .e.g `ota`, then we have to update SDK. How we can prevent that?

Reply:
yes we need to update SDK if we want to add filter for `ota`

Reply:
what do you mean by `how we can prevent that?`? do you mean that you want to support `ota` even though we aren't using it?

Reply:
i think he means to make providerType filter generic and dynamic, so when new filter comes along no need to change code. not sure if it's possible though

Reply:
Oh I see, yup it's possible